server/base/src/main/java/org/apache/accumulo/server/util/PropUtil.java

```diff
@@ -65,6 +65,11 @@ protected static void validateProperties(final ServerContext context,
         }
         throw new IllegalArgumentException(exceptionMessage + propStoreKey + " name: "
             + prop.getKey() + ", value: " + prop.getValue());
+      } else if (prop.getKey().equals(Property.TABLE_CLASSLOADER_CONTEXT.getKey())) {
+        if (!ClassLoaderUtil.isValidContext(prop.getValue())) {
```

Comment:
The else block can be removed and the indent promoted up one level to simplify this. Also the two if conditions can be combined with `&&`

Reply:
Completed in 9272c0f